### DISCUSSION TITLE
[Release] Super Zombie Bros

### DISCUSSION THREAD

**Discussion by notrickzdumbo** [nzp-team/nzportable#307] [Map Releases]
Here it is. A Super Mario Bros map for NZP.
Enjoy! And expect more maps from me :D

**Notes**
- There are two levels to the map. The ground section and the underground section
- It has Pack a Punch and all perks
- There is a buyable ending
- Games start a bit slow due to the size of the map but then gets quick once they start sprinting :)

![fte-20230630152821-0](https://github.com/nzp-team/nzportable/assets/98246579/b0fc17a7-d495-4f25-9aad-47a83ae36d90)
![fte-20230630152737-0](https://github.com/nzp-team/nzportable/assets/98246579/5dd8c1ec-da83-4e15-b98a-b0dd5ecd86bb)

**Changelog**
- V2
Added menu picture and description. Should fix crash on vita

### **WARNING!!!**
Unfortunately as of right now it only works on PC :(
I will fix it as soon as I can for v3

**Comment by MotoLegacy** [comment]
super fun to play through :^)

**Comment by jack5github** [comment]
Some of the player spawn points are stuck inside the walls, preventing this map from being played with 3 or more players.

**Reply by MotoLegacy** [reply]
This is a bug on my end.

**Comment by tysonbulls64-hash** [comment]
PLEASEPLEASEPLEASE UPDATE FOR VITAAAAAAA

**Comment by tysonbulls64-hash** [comment]
plsss i wanna play this on ps vita

**Comment by ProfessorZIRE** [comment]
Best custom nzp map I've found, chef's kiss
